database/db.go

```diff
@@ -0,0 +1,158 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+	"go_final_project/service/model"
+	"strings"
+	"time"
+)
+
+type DBStorage struct {
+	Client *sql.DB
+}
+
+func NewDBStorage(db *sql.DB) *DBStorage {
+	return &DBStorage{Client: db}
+}
+
+func (db *DBStorage) CreateTableScheduler() error {
+	createTableScheduler := `CREATE TABLE IF NOT EXISTS scheduler (
+			id INTEGER PRIMARY KEY AUTOINCREMENT,
+			date CHAR(8) NOT NULL DEFAULT "",
+			title VARCHAR(256) NOT NULL DEFAULT "",
+			comment VARCHAR(256) NOT NULL DEFAULT "",
+			repeat VARCHAR(128) NOT NULL DEFAULT ""
+		);`
+
+	_, err := db.Client.Exec(createTableScheduler)
+	if err != nil {
+		return fmt.Errorf("Ошибка создания таблицы scheduler в базе данных: %s", err)
```

Comment:
С большой буквы, на русском, так еще и %s
```suggestion
		return fmt.Errorf("error while creating table scheduler: %w", err)
```